shared/src/main/java/net/blay09/mods/balm/api/event/server/ServerBeforeStartingEvent.java

```diff
@@ -0,0 +1,17 @@
+package net.blay09.mods.balm.api.event.server;
+
+import net.blay09.mods.balm.api.event.BalmEvent;
+import net.minecraft.server.MinecraftServer;
+
+/** Fired before any levels are loaded */
+public class ServerBeforeStartingEvent extends BalmEvent {
```

Comment:
I think this should be called either `ServerStartingEvent` or `ServerAboutToStartEvent`